### PR TITLE
fix: give a bunch of list renders keys

### PR DESCRIPTION
**Pull request by JFrankfurt**

### CODE REVIEW

File: src/nft/components/explore/CarouselCard.tsx [hunk 1]
```diff
@@ -223,13 +223,13 @@ export const CarouselCard = ({ collection, onClick }: CarouselCardProps) => {
...
             <HeaderRow>{gqlCollection.marketplaceCount?.reduce((acc, cur) => acc + cur.count, 0)} Listings</HeaderRow>
-            {MARKETS_TO_CHECK.map((market) => {
+            {MARKETS_TO_CHECK.map((market, index) => {
               const marketplace = gqlCollection.marketplaceCount?.find(
                 (marketplace) => marketplace.marketplace === market
               )
               return (
                 <MarketplaceRow
-                  key={'trendingCollection' + collection.address}
+                  key={`CarouselCard-key-${collection.address}-${index}`}
```
Comment (vm) [inline src/nft/components/explore/CarouselCard.tsx:232]: this should be keyed on market not index? how can there be duplicates

Comment (JFrankfurt) [reply src/nft/components/explore/CarouselCard.tsx]: done

**Review by vm — APPROVED** [review]
i do think we should filter out any undefined market places instead of needing the conditionals

**Comment by JFrankfurt** [comment]
@vm yeah that makes sense. I'll do that before I merge

**Review by lynnshaoyu — APPROVED** [review]

**Pull request merged**